pytorch_lightning/trainer/logging.py

```diff
@@ -73,8 +74,6 @@ def log_metrics(self, metrics, grad_norm_dic, step=None):
         # log actual metrics
         if self.is_global_zero and self.logger is not None:
             self.logger.agg_and_log_metrics(scalar_metrics, step=step)
-            self.logger.save()
```

Comment:
How is this related to https://github.com/PyTorchLightning/pytorch-lightning/issues/3398#issuecomment-691011447?
@SkafteNicki Maybe you know something about this also?